### PR TITLE
NEP141-Locker: parse fee from msg

### PR DESCRIPTION
**Pull request by karim-en**

### CODE REVIEW

File: near/omni-types/src/lib.rs [hunk 1]
```diff
@@ -193,6 +193,12 @@ impl fmt::Display for NearRecipient {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct InitTransferMsg {
+    pub recip: OmniAddress,
```
Comment (kiseln) [inline near/omni-types/src/lib.rs:198]: Why not `recipient`?

**Review by olga24912 — APPROVED** [review]

File: near/omni-types/src/lib.rs (hunk 1 above)
Comment (karim-en) [reply near/omni-types/src/lib.rs:198]: The idea was to reduce the message size so the transaction could be signed using the ledger.
Anyway, let me rename it to `recipient` and return it to `recip` if it is needed

Comment (karim-en) [reply near/omni-types/src/lib.rs]: 209b11bca92e4f4b4a4ba7b9a1a6f619745ac6aa

**Review by kiseln — APPROVED** [review]

File: near/omni-types/src/lib.rs (hunk 1 above)
Comment (kiseln) [reply near/omni-types/src/lib.rs]: New ledger screen is pretty big it seems

Comment (karim-en) [reply near/omni-types/src/lib.rs]: @kiseln it isn't about the screen, but about the payload limits for a near wallet on ledger

**Pull request merged**